recipes/stages/classification/class_incr_multilabel.yaml

```diff
@@ -0,0 +1,37 @@
+_base_: [
```

Comment:
I couldn't find where this file is used. Is this file used only for experiment purpose?